packages/api/tests/endpoints/collections/create-collection.js

```diff
@@ -523,5 +523,5 @@ test.serial('POST does not write to SNS if writing to PostgreSQL fails', async (
     WaitTimeSeconds: 10,
   });
 
-  t.is(Messages.length, 0);
+  t.is(Messages, undefined);
```

Comment:
It's been back and forth, but localstack returns `undefined` if there is no message in the queue.  It's conform with aws-sdk doc.